reverse_string/reverse.go

```diff
@@ -1,7 +1,7 @@
 package reverse_string
 
 func ReverseString(input string) string {
-	runes := []rune(input)
+	rune := []rune(input)
```

Comment:
Похоже, в названии переменной пропущена `s`

```suggestion
	runes := []rune(input)
```

Reply:
Это был тестовый PR, чтобы проверить, что статус FAILED если билд или тесты не прошли

Reply:
> Желательно, чтобы PR содержал все файлы по задаче

Я просто в начале пушил все в мастер, без PRов